ansible/roles/mongoose/templates/mongoose.service.j2

```diff
@@ -9,11 +9,11 @@ OnFailure=failure-notificator@%n.service
 User=mongoose
 Group=mongoose
 WorkingDirectory=/var/www/mongoose/undead-mongoose
-ExecStartPre=/nix/var/nix/profiles/default/bin/nix-shell --run './manage.py collectstatic --clear --no-input'
-ExecStart=/nix/var/nix/profiles/default/bin/nix-shell --run  'gunicorn undead_mongoose.wsgi \
+ExecStartPre=uv run manage.py collectstatic --clear --no-input
```

Comment:
In systemd services it is recommended to provide full paths to the executable. This, because it is not easy to ensure that a shell with proper PATH initialisation is used

Reply:
Oh you're right, I thought I had these changes pushed already.